snapcraft.yaml

```diff
@@ -10,15 +10,20 @@ base: core24
 assumes: [kernel-assets]
 platforms:
   rpi:
-    build-on: arm64
-    build-for: arm64
+    build-on: armhf
+    build-for: armhf
   rpi-amd64:
     build-on: amd64
-    build-for: arm64
+    build-for: armhf
+  rpi-arm64:
+    build-on: arm64
+    build-for: armhf
 
 package-repositories:
   - type: apt
     ppa: snappy-dev/image
+  - type: apt
+    ppa: juergh/pi-kernel-armhf
```

Comment:
Is this PPA excluded from Launchpad automatic deleted files collection for archival purposes?

(cc: @vorlonofportland @mwhudson )